module-api/src/main/resources/application.yml

```
@@ -12,3 +12,7 @@ spring:
   thymeleaf:
     prefix: classpath:/templates/
     suffix: .html
+  data:
+    redis:
+      host: localhost
+      port: 6379
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)